doc/usage.rst

```diff
@@ -60,80 +60,81 @@ On the bottom left is a `google` icon.
 .. image:: google.png
 
 If you click on the `google` icon, you will be able to see a bigger version
-of the image, allowing to see it in more detail.
+of the image, allowing you to see it in more detail.
```

Comment:
the sentence starts by ``you`` already, so I am not sure adding another one here makes sense :)